app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+    # Validation for unique and valid email addres
```

Comment:
As we discussed we can skip this commend

Reply:
Changed as well